maestro-cli/src/main/java/maestro/cli/runner/MaestroCommandRunner.kt

```diff
@@ -89,6 +90,9 @@ object MaestroCommandRunner {
         }
 
         refreshUi()
+        if (analyze) {
+            ScreenshotUtils.takeDebugScreenshotByCommand(maestro, debugOutput, CommandStatus.PENDING)
+        }
```

Comment:
Why an extra screenshot here?

Reply:
Btw I added the suggestion to filter similar images and not it's ignoring the unnecessary cookies programmatically